core/src/main/scala/cats/data/NonEmptyVector.scala

```diff
@@ -115,6 +115,12 @@ final class NonEmptyVector[+A] private (val toVector: Vector[A])
    */
   def prepend[AA >: A](a: AA): NonEmptyVector[AA] = new NonEmptyVector(a +: toVector)
 
+  /**
+   * Prepend a `Vector` to this, producing a new `NonEmptyVector`.
+   */
+  def prependVec[AA >: A](vector: Vector[AA]): NonEmptyVector[AA] =
```

Comment:
What do you think of naming this `prependVector` for consistency, on the model of the existing `prependLazyList` and `prependChain`?

Reply:
Sounds good. Happy to rename it so it's consistent with similar constructs. 👍

Reply:
Thanks much!